yoda/llava_data_prep/src/generate_tables.py

```diff
@@ -0,0 +1,45 @@
+# Helper utility script based on Fenglu Hong @SambaNova Systems' work for table OCR fine tuning.
+# Use to quickly iterate on table formatting when building templates.
+
+import os
+import sys
+
+current_dir = os.getcwd()
+kit_dir = os.path.abspath(os.path.join(current_dir, "../.."))
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
+
+sys.path.append(kit_dir)
+sys.path.append(repo_dir)
+
+from yoda.llava_data_prep.src.table_utils import TableTools # type: ignore
+DATA_DIRECTORY = os.path.join(kit_dir,"synthetic_data")
+
+table_tools = TableTools()
+
+synth_tables: dict = {}
+
+columns: str = "\\begin{tabular}{| l c c r c c r |}\n\\hline\n"
```

Comment:
Should these 2 literal strings perhaps be in a config somewhere? I assume they change with different types of table to be generated?